app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1> All Rooms: </h1>
+
+<% @rooms.each do |room| %>
+  <h3> Suite: <%= room.suite %> </h3>
+  <h3> Nightly Rate: <%= room.rate %> </h3>
+  <h3> Hotel: <%= room.hotel.name %> </h3>
+  <div id = "count-<%= room.id %>">
+  <h3> Total Number of Guests: <%= room.room_guests.count %> </h3>
+  </div>
+  <br>
+<% end %>
```

Comment:
I mentioned this already in your `Room` model, but by doing `<%= room.room_guests.count %>`, we are actually performing ActiveRecord in the view, thus breaking MVC conventions. You want to move this to the `Room` model.